### PR TITLE
Add source files for jQuery and dataTables.

### PR DESCRIPTION
**Pull request by saimn**
+ Ref #4330: source files are needed for Debian packaging.
+ Use the non-minified version for the dataTables css as the gain is negligible.

### CODE REVIEW

**Comment by saimn** [comment]
For the record: DataTables 1.10.10 is available but I have added the source for 1.10.9 which correspond to the minified version already present in astropy.
Version 1.10.10 bring some changes for the AMD module definition which is used by the show in notebook feature, so I think it is better to keep this version for later.
See http://datatables.net/feeds/releases.xml for the changelog (no really practical by it is the only link I found).

**Comment by embray** [comment]
Thanks for the quick turn-around on this @saimn !

**Comment by eteq** [comment]
Great, thanks @saimn!

One suggestion: maybe add a ``README`` to the ``extern/js`` directory that explains that the minified files are the ones that are actually *used*, but the non-minified versions are there so that the full source code is present in the distribution?  That way hopefully we can avoid this happening again.

Also, @olebole (and maybe @sergiopasra if you have a moment), can you confirm that this now confirms to the requirements for packaging?

**Comment by olebole** [comment]
Yes, thanks! This is what I needed for packaging.

**Comment by saimn** [comment]
> One suggestion: maybe add a README to the extern/js directory that explains that the minified files are the ones that are actually used, but the non-minified versions are there so that the full source code is present in the distribution? That way hopefully we can avoid this happening again.

Yes, good idea, I'm going to add this.

**Comment by astrofrog** [comment]
@saimn - thanks! Would it be possible to do this today so that we can include it in the release candidate?

**Comment by saimn** [comment]
@astrofrog oh yes, of course, I take a look.

**Comment by saimn** [comment]
Actually there is already a [README](https://github.com/astropy/astropy/blob/master/astropy/extern/README.rst) in extern/ , with a note about JQuery / DataTables for packagers, so it make sense to a note for developers about (non-)minified files, right ?

**Comment by saimn** [comment]
Done, is the formulation ok ?

File: astropy/table/jsviewer.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class Conf(_config.ConfigNamespace):
         'The URL to the jquery datatables library.')
 
     css_urls = _config.ConfigItem(
-        ['https://cdn.datatables.net/1.10.9/css/jquery.dataTables.min.css'],
+        ['https://cdn.datatables.net/1.10.9/css/jquery.dataTables.css'],
```
Comment (astrofrog) [inline astropy/table/jsviewer.py]: I thought we were using the minified ones? Why change the default? (and same comment below)

Comment (eteq) [inline astropy/table/jsviewer.py]: For the CSS, I think @saimn noted above that the minified version is not much smaller than the regular one so there seems little point in having the minified version at all. The source, though, should be the mins (which are quite a bit smaller/more optimized)

Comment (astrofrog) [inline astropy/table/jsviewer.py]: Ah ignore me, didn't realize this was the CSS not the JS

**Comment by astrofrog** [comment]
:+1:  from me

**Comment by eteq** [comment]
Merged!  And backported to v1.1.x in 1f268d5d358ab8ba861bb3ad3b9b7ba6572e8dca

**Pull request merged**